github-check-releases/README.md

```diff
@@ -0,0 +1,117 @@
+# GitHub Check Releases
+
+This directory contains tooling to enable us to check GitHub releases programmatically, and validate them against the
+version(s) of the product(s) we use in our code example test suite(s).
+
+This is a simple PoC with three main components:
+
+- A [JSON file](repo-details.json) that contains the list of products, GitHub repo information, and test suite versions.
+- It uses [octokit](https://github.com/octokit/octokit.js) to get release data for those MongoDB products from GitHub, and compare it against the version 
+  in our JSON file.
+- It uses [axios](https://github.com/axios/axios) with the Jira API to automatically create tickets to update test suite 
+  versions.
+
+## Add or change products and test suite versions
+
+This project pulls data from [repo-details.json](repo-details.json) to check the release versions for the MongoDB
+products we use in our code example test suites. It deserializes the data using `RepoDetails` declared in the
+[RepoDetails file](RepoDetails.js) to map the owner and repo name for a given product whose version we want to check.
```

Comment:
```suggestion
products we use in our code example test suites.
```
this seems in-the-weeds for a README. maybe put this in the class file as a comment?